### PR TITLE
add current_events_magnifier recipe

### PR DESCRIPTION
**Pull request by JeremyQuo**
Describe your pull request here
Add the current_events_magnifier to showcase ONT data
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by JeremyQuo** [comment]
@bioconda-bot please add label

File: recipes/current_events_magnifier/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+test:
+  imports:
+    - current_events_magnifier
+
+about:
+  home: "https://github.com/lrslab/current_events_magnifier"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (bgruening) [inline recipes/current_events_magnifier/meta.yaml:46]: This here is missing and please add the doc_url or clean up the recipe a bit. Thanks.

File: recipes/current_events_magnifier/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  sha256: 2719b11ab8046909e6b34907f0a0c3368b22c45df5f4769746dd3a47ebef73b6
+
+build:
+  number: 0
+  noarch: generic
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - h5py ==3.8.0
```
Comment (bgruening) [inline recipes/current_events_magnifier/meta.yaml:19]: you can remove everything but pip and python from the host requiremen

File: recipes/current_events_magnifier/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2719b11ab8046909e6b34907f0a0c3368b22c45df5f4769746dd3a47ebef73b6
+
+build:
+  number: 0
+  noarch: generic
```
Comment (bgruening) [inline recipes/current_events_magnifier/meta.yaml:14]: this looks like a noarch: python recipe

File: recipes/current_events_magnifier/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "A sample tool designed to visualize the features that distinguish between two groups of ONT data at the site level.                It supports two re-squiggle pipeline(Tombo and f5c)."
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (bgruening) [inline recipes/current_events_magnifier/meta.yaml:53]: this is also not correct :)

**Review by bgruening — CHANGES_REQUESTED** [review]
Thanks, a few comments inline.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by dpryan79** [comment]
It seems the package name has changed. I can't easily rename the directory to match, so you'll need to do that.

**Pull request closed without merging**